streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java

```diff
@@ -540,18 +534,18 @@ private void checkCacheMetrics(final String builtInMetricsVersion) {
             .collect(Collectors.toList());
         checkMetricByName(
             listMetricCache,
-            builtInMetricsVersion.equals(StreamsConfig.METRICS_LATEST) ? HIT_RATIO_AVG : HIT_RATIO_AVG_BEFORE_24,
-            builtInMetricsVersion.equals(StreamsConfig.METRICS_LATEST) ? 3 : 6 /* includes parent sensors */
+            builtInMetricsVersion.equals("latest") ? HIT_RATIO_AVG : HIT_RATIO_AVG_BEFORE_24,
```

Comment:
I understand we want to hide the built-in metrics, but since we are only replacing `METRICS_LATEST` with the string value, would we want to consider leaving `METRICS_LATEST` in?  I could very well be missing something here.

Reply:
I moved the values of the configuration temporarily to `StreamsMetricsImpl`. So, we can use them in the internal code. The only cons is that now integration test `MetricsIntegrationTest` depends on `StreamsMetricsImpl` which actually is an internal class. It is a trade-off.

Reply:
Ah ok, I get it.  I'll leave it up to you to do what you think is best.